src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.pow(side, 2);
```

Comment:
Consider defining '2' as a constant with a clear name to indicate its purpose. Magic numbers can make code less readable and harder to maintain.